src/ducks/GainersAndLosers/GainersLosersPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import { graphql } from 'react-apollo'
+import { Panel } from '@santiment-network/ui'
+import GainersLosersTable from './GainersAndLosersTable'
+import { Tabs } from '@santiment-network/ui'
+import { allProjectsGQL } from '../../pages/Projects/allProjectsGQL'
+import styles from './GainersAndLosersPage.module.scss'
+import { mapItemsToKeys } from '../../utils/utils'
+
+const statusMapping = {
+  Gainers: 'GAINER',
+  Losers: 'LOSER'
+}
+
+const timeWindowMapping = {
+  '24h': '2d',
+  '2w': '15d'
+}
+
+class GainersAndLosersPage extends Component {
+  state = {
+    timeWindow: '2d',
+    status: 'GAINER'
```

Comment:
`statusMapping` and `timeWindowMapping` created above, while the state is constructed with literal values.